sidebars.js

```diff
@@ -978,6 +978,9 @@ module.exports = {
     {
       "Zeebe Gateway": ["self-managed/zeebe-gateway-deployment/zeebe-gateway"],
     },
+    {
+      "Web Modeler (Beta)": ["self-managed/web-modeler/installation"],
+    },
```

Comment:
FYI: There's not a lot of content here, but I thought for the beta release it would be fine. Later we could at least add a dedicated page on configuration, for example.